cvat/apps/engine/static/engine/js/annotationUI.js

```diff
@@ -609,11 +609,12 @@ function saveAnnotation(shapeCollectionModel, job) {
         'points count': totalStat.points.annotation + totalStat.points.interpolation,
     });
 
-    let exportedData = shapeCollectionModel.export();
-    let annotationLogs = Logger.getLogs();
+    const exportedData = shapeCollectionModel.export();
+    shapeCollectionModel.reset_state();
+    const annotationLogs = Logger.getLogs();
 
     const data = {
-        annotation: exportedData,
+        annotation: JSON.stringify(exportedData),
```

Comment:
What is the reason to send it as string?

Reply:
I think cause of issue was hash function, that used early. I've not changed request format for backward capability and suggest to change it in new MR.